auth/src/main/res/layout-land/fui_auth_method_picker_layout.xml

```diff
@@ -22,16 +22,17 @@
         app:layout_constraintEnd_toStartOf="@+id/container"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintBottom_toTopOf="@+id/main_tos_and_pp"
+        android:layout_marginTop="@dimen/fui_field_padding_vert"
+        android:layout_marginBottom="@dimen/fui_field_padding_vert"
         tools:ignore="ContentDescription" />  <!-- TODO remove once the bug is fixed: https://issuetracker.google.com/issues/38281866-->
 
     <TextView
         android:id="@+id/main_tos_and_pp"
         style="@style/FirebaseUI.Text.BodyText"
         android:layout_width="0dp"
-        android:layout_height="0dp"
+        android:layout_height="wrap_content"
```

Comment:
I wanted to do that too, but then this happens. Maybe you know more ConstraintLayout jitsu? 😆 

![screenshot_20180925-103634](https://user-images.githubusercontent.com/9490724/46032232-e47ce400-c0af-11e8-97a1-a8c261050db8.png)

Reply:
Hmmm.  I probably need to mess with the vertical chain type then.  All the "padding" on the ToS/PP text isn't actually padding, it's a result of the view bounds stretching to meet its constraints.